app/gateway/cp-outage.md

```diff
@@ -0,0 +1,209 @@
+
+## Configure data plane resilience 
+
+Data plane resilience is managed in the [`kong.conf`](/gateway/manage-kong-conf/) configuration file by the following parameters: 
+
+```
+cluster_fallback_config_import: on
+cluster_fallback_config_storage: $STORAGE_ENDPOINT
+cluster_fallback_config_export = off
+```
+
+Avoid configuring both the data plane and control plane to `export` a configuration. Assuming that they are both configured with the correct level of authentication, the data plane will write to the storage volume first, and then the control plane will overwrite the configuration.
+
+The following table provides details about these parameters:
+
+<!--vale off-->
+{% kong_config_table %}
+config:
+  - name: cluster_fallback_config_import
+    description: Fetches the fallback configuration from the URL passed in `cluster_fallback_config_storage` if the CP is unreachable. This should only be enabled on the data plane.
+  - name: cluster_fallback_config_storage
+  - name: cluster_fallback_config_export
+    description: This parameter allows you to upload the configuration to the storage volume. This should only be enabled on the backup node.
+{% endkong_config_table %}
+<!--vale on-->
+
+In addition, you'll also need to configure settings based on the S3-compatible storage type you're using.
+
+### Amazon S3 storage
+
+In this setup, you need to designate one backup node. 
+The backup node must have read and write access to the S3 bucket, and the data plane nodes that are provisioned must have read access to the same S3 bucket.
+The backup node is responsible for communicating the state of the {{site.base_gateway}} `kong.conf` configuration file from the control plane to the S3 bucket.
+
+Nodes are initialized with fallback configs via environment variables, including `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_DEFAULT_REGION`. 
+If you're associating this with an IAM role and if the backup node doesn't reside on the AWS platform, you may also need to use the `AWS_SESSION_TOKEN` environment variable. 
+
+
+{:.important}
+> We don't recommend using backup nodes to proxy traffic. The backup job enlarges the attack surface of a proxying data plane and contributes significantly to the P99 delay. You need to know the risk if you want to deploy a node this way, 
+{% if_version lte:3.5.x %} and a data plane acting as a backup node cannot be provisioned with backup configurations.{% endif_version %}{% if_version gte:3.6.x %}and the data plane needs to be at least `3.6.0.0` to be provisioned with backup configuration when it's configured as a backup node. 
+Although a single backup node is sufficient for all deployments, you can also configure additional backup nodes. A leader election algorithm selects one node from the group of designated backup nodes to do the backup job.
+{% endif_version %}
```

Comment:
```suggestion
```
I think it should be warning or danger and we aren't planning on porting over the version stuff. at least I don't think so.